test/axes/timeAxisTests.ts

```diff
@@ -149,6 +149,60 @@ describe("TimeAxis", () => {
     svg.remove();
   });
 
+  it("tick labels do not overlap with tick marks in top orientation", () => {
+     let svg = TestMethods.generateSVG(400, 100);
+      scale = new Plottable.Scales.Time();
+      scale.domain([new Date("2010-01-01"), new Date("2014-01-01")]);
+      axis = new Plottable.Axes.Time(scale, "top");
+
+      function checkTierDisplayPosition (tierDisplayPositions: any[]) {
+        axis.tierLabelPositions(tierDisplayPositions);
+        axis.renderTo(svg);
+
+        let style = (<any>axis)._element.append("style");
+        style.attr("type", "text/css");
+        style.text(".plottable .axis.time-axis text { font-family: Arial; }");
+
+        let tickMarks = d3.selectAll(`.${Plottable.Axis.TICK_MARK_CLASS}:not(.${Plottable.Axis.END_TICK_MARK_CLASS})`);
+        assert.operator(tickMarks.size(), ">=", 1, "There are more than one tick marks in the test");
+
+        let tickLabels = d3.selectAll(`.${Plottable.Axis.TICK_LABEL_CLASS}`)
+        .filter(function(d: Element, i: number){
+          return window.getComputedStyle(this).visibility !== "hidden";
+        });
+        assert.operator(tickLabels.size(), ">=", 1, "There are more than one tick labels in the test");
+
+        function myclientRectsOverlap(clientRectA: ClientRect, clientRectB: ClientRect) {
```

Comment:
rename to `clientRectsOverlap`. Also can we use the use `Utils.DOM.clientRectsOverlap()` or there is a issue with the Pixel Close To?